deploy/vagrant/provision/roles/ufs_gcs/files/config_alluxio.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+sed -i "s|^export ALLUXIO_UNDERFS_ADDRESS=.*|export ALLUXIO_UNDERFS_ADDRESS=gs://${GCS_BUCKET}|g" /alluxio/conf/alluxio-env.sh
```

Comment:
if there are other occurrences of this pattern in the deploy module, they should be removed